app/tests/profiles_tests/test_views.py

```diff
@@ -179,3 +180,74 @@ def test_organization_update(self, client):
         )
 
         assert org1.title not in response.content.decode()
+
+
+@pytest.mark.parametrize(
+    "viewname,subscription_attr",
+    (
+        ("newsletter-unsubscribe", "receive_newsletter"),
+        ("notification-unsubscribe", "receive_notification_emails"),
+    ),
+)
+@pytest.mark.django_db
+def test_one_click_unsubscribe(client, settings, viewname, subscription_attr):
+    # Override the celery settings
+    settings.task_eager_propagates = (True,)
+    settings.task_always_eager = (True,)
```

Comment:
Why is Celery required for the test?

Reply:
Because we are creating/updating a verification user set if there is a user mismatch and I'm calling the asynchronous task `update_verification_user_set` to do that.